pkg/crc/api/client/client.go

```diff
@@ -37,7 +37,7 @@ func (c *Client) Version() (VersionResult, error) {
 
 func (c *Client) Status() (ClusterStatusResult, error) {
 	var sr = ClusterStatusResult{}
-	body, err := c.sendGetRequest("/status")
+	body, err := c.sendGetRequest("/status", nil)
```

Comment:
do we really need a `sendGetRequest(path, nil)`,

why not an override `sendGetRequest(path)` method that internally calls `sendGetRequest(path, nil)`

Reply:
@gbraad No, that would be too much override, I am already adding a helper function for all type of request, adding another helper on top of Get doesn't make sense, I would be more explicit here for testing purpose also.

Reply:
> No, that would be too much override,

a default parameter? Not sure what you mean with too much override. it would also be clear, or could be naming... anyway, a fix is 'another helper' is what I was looking for.